node/src/chain_spec/turing.rs

```diff
@@ -160,6 +160,7 @@ fn testnet_genesis(
 		treasury: Default::default(),
 		valve: ValveConfig { start_with_valve_closed: false, closed_gates: pallet_gates_closed },
 		vesting: VestingConfig { vesting_schedule },
+		asset_registry: Default::default(),
```

Comment:
this will need to change in order for https://github.com/OAK-Foundation/OAK-blockchain/pull/269 to work. Does not need to be in this PR though.